bool home
bool position_uncertain